layers/generate_layers.py

```diff
@@ -65,6 +75,64 @@ def generatePolygonAmenity(args, amenity, filename):
     run(["tippecanoe", f"{filename}.geojson", "-o", f"{filename}.pmtiles"])
 
 
+def makeMRN():
+    # Remove files from any previous run
+    try:
+        os.remove("Major_Road_Network_2018_Open_Roads.zip")
+        os.remove("mrn.pmtiles")
+        shutil.rmtree("mrn")
+    except:
+        pass
+
+    # Get the shapefile
```

Comment:
There are Python libraries that wrap some of the commands below, but I don't think there's a particular advantage to using them (and external dependencies, especially on things like GDAL wrapper libraries, are a headache requiring us to pick a Python build tool). All the data prep scripts are effectively shell scripts so far, with a bit of Python to transform JSON data, so I'm continuing with this simple approach for now.